package.json

```diff
@@ -76,6 +75,6 @@
     "protractor": "~5.1.2",
```

Comment:
can you try to bump to latest version?

Reply:
also can you try `~2.53.39` for selenium and see if the error in test suite gets fixed